QuizApp/Quiz/QuizViewController.swift

```diff
@@ -14,12 +36,125 @@ class QuizViewController: UIViewController {
+        titleLabel.text = "PopQuiz"
+        titleLabel.textColor = .white
+
+        segmented.addTarget(self, action: #selector(newCategoryChosen), for: .valueChanged)
+        segmented.layer.cornerRadius = 10
+        segmented.selectedSegmentIndex = 0
+        let titleAttributes = [NSAttributedString.Key.foregroundColor: UIColor.white]
+        segmented.setTitleTextAttributes(titleAttributes, for: .normal)
+        segmented.setTitleTextAttributes(titleAttributes, for: .selected)
+        segmented.apportionsSegmentWidthsByContent = true
+        segmented.selectedSegmentTintColor = UIColor(red: 0.453, green: 0.308, blue: 0.637, alpha: 1)
+        segmented.backgroundColor = UIColor(red: 0.154, green: 0.185, blue: 0.463, alpha: 0.5)
+
+        quizListCollectionView.layer.cornerRadius = 7
+        configureCollectionView()
+    }
+
+    func defineLayoutForViews() {
+        gradientBg.frame = view.bounds
+        gradientBg.locations = [0, 1]
+
+        titleLabel.snp.makeConstraints {
+            $0.centerX.equalToSuperview()
+            $0.top.equalTo(self.view.safeAreaLayoutGuide.snp.top).inset(-30)
+            $0.height.equalTo(30)
+        }
+
+        segmented.snp.makeConstraints {
+            $0.top.equalTo(titleLabel.snp.bottom).offset(20)
+            $0.leading.equalTo(self.view.safeAreaLayoutGuide.snp.leading).offset(10)
+            $0.trailing.equalTo(self.view.safeAreaLayoutGuide.snp.trailing).inset(10)
+            $0.height.equalTo(30)
+        }
+
+        quizListCollectionView.snp.makeConstraints {
+            $0.top.equalTo(segmented.snp.bottom).offset(20)
+            $0.leading.equalTo(self.view.safeAreaLayoutGuide.snp.leading).offset(10)
+            $0.trailing.equalTo(self.view.safeAreaLayoutGuide.snp.trailing).inset(10)
+            $0.bottom.equalTo(self.view.safeAreaLayoutGuide.snp.bottom).inset(5)
+        }
+    }
+
+}
+
+extension QuizViewController: UICollectionViewDataSource {
+
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return quizViewModel.mockQuizzes.count
+    }
+
+    func collectionView(_ collectionView: UICollectionView,
+                        cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: QuizListCollectionViewCell.id,
+                                                      for: indexPath) as? QuizListCollectionViewCell
+        cell!.set(for: quizViewModel.mockQuizzes[indexPath.row])
+        return cell!
+    }
+
+    func configureCollectionView() {
+        quizListCollectionView.register(QuizListCollectionViewCell.self,
+                                        forCellWithReuseIdentifier: QuizListCollectionViewCell.id)
+        quizListCollectionView.backgroundColor = .clear
+        quizListCollectionView.delegate = self
+        quizListCollectionView.dataSource = self
+        quizListCollectionView.isScrollEnabled = true
```

Comment:
```
quizListCollectionView.backgroundColor = .clear
quizListCollectionView.isScrollEnabled = true
```
are style sonfigurations and should be called in ` func styleViews()` but other ones aren't style configurations so call them separately, in `viewDidLoad`.